invenio/templates/configurations/invenio.yaml

```diff
@@ -10,10 +10,16 @@ data:
   INVENIO_CACHE_REDIS_URL: 'redis://{{ include "redis.host_name" . }}:6379/0'
   INVENIO_CELERY_RESULT_BACKEND: 'redis://{{ include "redis.host_name" . }}:6379/2'
   INVENIO_COLLECT_STORAGE: flask_collect.storage.file
-  INVENIO_DATACITE_ENABLED: "{{ .Values.invenio.datacite.enabled }}"
   {{ if .Values.invenio.datacite.enabled }}
   INVENIO_DATACITE_PREFIX: "{{ .Values.invenio.datacite.prefix }}"
-  INVENIO_DATACITE_TEST_MODE: "{{ .Values.invenio.datacite.test_mode }}"
+  INVENIO_DATACITE_ENABLED: "True"
```

Comment:
minor: is there a way to have a small utility function/macro to return the correct boolean?